packages/grafana-runtime/src/utils/plugin.ts

```diff
@@ -34,7 +23,7 @@ export const SystemJS = window.System;
 export async function loadPluginCss(options: PluginCssOptions): Promise<System.Module | void> {
   try {
     const cssPath = config.bootData.user.theme === 'light' ? options.light : options.dark;
-    return SystemJS.import(cssPath);
+    return window.System.import(cssPath);
```

Comment:
what's the risk of `System` not existing in `window` at this moment? Would it bee too much to create a `getSystemJs` that will return a promise until `window.System` is defined?

Reply:
This function is called by old angular plugins which can only execute it once it's been loaded by Systemjs which makes me think it's safe. 🐔 🥚 🤔